app/components/page_settings_summary_component/view.html.erb

```diff
@@ -1,73 +1,102 @@
-                 visually_hidden_text: t("page_settings_summary.answer_type")
-      ) end %>
+  <%= summary_list.with_row do |row| %>
+    <%= row.with_key(text: t("page_settings_summary.answer_type")) %>
+    <%= row.with_value(text: t("helpers.label.page.answer_type_options.names.#{@draft_question.answer_type}")) %>
+    <%= row.with_action(text: t("page_settings_summary.change"),
+                    href: @change_answer_type_path,
+                    visually_hidden_text: t("page_settings_summary.answer_type")) %>
+  <% end %>
 
   <% if show_selection_settings_summary %>
-    <%= summary_list.with_row do |row|
-          row.with_key(text: t("page_settings_summary.selection.options"))
-          row.with_value(text: show_selection_options)
-          row.with_action(text: t("page_settings_summary.change"),
+    <%= summary_list.with_row do |row| %>
+      <%= row.with_key(text: t("page_settings_summary.selection.options")) %>
+      <%= row.with_value do %>
+        <%= row.with_value do %>
+          <p>
+            <%= t("page_settings_summary.selection.options_count", number_of_options: selection_options.length) %>
+          </p>
+          <ul class="govuk-list govuk-list--bullet">
+            <% selection_options.each do |option| %>
+              <li><%= option[:name] %></li>
+            <% end %>
+          </ul>
+        <% end %>
+      <% end %>
+      <%= row.with_action(text: t("page_settings_summary.change"),
                     href: @change_selections_options_path,
-                    visually_hidden_text: t("page_settings_summary.selection.options")) end %>
-    <%= summary_list.with_row do |row|
-          row.with_key(text: t("page_settings_summary.selection.only_one_option"))
-          row.with_value(text: answer_settings[:only_one_option] == "true" ? t("selections_settings.yes") : t("selections_settings.no"))
-          row.with_action(text: t("page_settings_summary.change"),
-                    href: @change_selections_only_one_option_path,
-                    visually_hidden_text: t("page_settings_summary.selection.only_one_option")) end %>
-    <%= summary_list.with_row do |row|
-          row.with_key(text: t("page_settings_summary.selection.include_none_of_the_above"))
-          row.with_value(text: @draft_question.is_optional ? t("selections_settings.yes") : t("selections_settings.no"))
-          row.with_action(text: t("page_settings_summary.change"),
+                    visually_hidden_text: t("page_settings_summary.selection.options")) %>
+    <% end %>
+    <% if @long_lists_enabled %>
+      <%= summary_list.with_row do |row| %>
+        <%= row.with_key(text: t("page_settings_summary.selection.how_many_selections")) %>
+        <%= row.with_value(text: answer_settings[:only_one_option] == "true" ? t("helpers.label.pages_long_lists_selection_type_input.only_one_option_options.true") : t("helpers.label.pages_long_lists_selection_type_input.only_one_option_options.false")) %>
```

Comment:
Up to you whether you think this is better but I think this is equivalent:
```suggestion
        <%= row.with_value(text: t("helpers.label.pages_long_lists_selection_type_input.only_one_option_options.#{answer_settings[:only_one_option]}")) %>
```

Reply:
Ah, I fogot about that!

Reply:
Added the test